test/bmi_calculator/core_test.clj

```diff
@@ -10,28 +10,15 @@
       1.0 {:x 100.0 :y 10.0}
       )))
 
-(defn get-input-helper
```

Comment:
Get rid of destructuring in `bmi-test` too

Tip: whenever receving feedback, search for more ocurrences of that issue